src/styles/index.scss

```diff
@@ -23,3 +23,4 @@ body {
 @import "./todoapp";
```

Comment:
It seems that you have added a new import for './transition' in the index.scss file. Make sure the corresponding transition.scss file exists and contains the necessary styles for the new todo features (adding and deleting todos).